Fix missing fsck support for exFAT devices

FacuM opened the pull request:
This pull request aims at fixing the missing support for fsck on exFAT SD cards (or any other external device).

Also, it introduces a couple of the following important packages that aren't compiled by default:

Terminal
WallpaperPicker

NTFS utilities are also bundled.

AdrianDC committed:
A6020: Add exFAT and NTFS filesystem tools for AOSP builds

Change-Id: Ib04d31a7069040ae6b801c4159927ac32ff044d3

adithya2306 commented:
Terminal and wallpicker is cool, the recorders are ugly (afaik) so no need of them, and messaging is now built by default

FacuM commented:
> Terminal and wallpicker is cool, the recorders are ugly (afaik) so no need of them, and messaging is now built by default

Just made the required changes, please take a look at this PR whenever you're free.

adithya2306 commented:
I've already added WallPicker and Terminal in https://github.com/AOSP-LEGACY/device_lenovo_A6020/commit/f16c24f22ec6dc03f05e44056fde4c1899b1135a
I'll merge the "A6020: Add exFAT and NTFS filesystem tools for AOSP builds" commit soon.

FacuM commented:
> I've already added WallPicker and Terminal in [f16c24f](https://github.com/AOSP-LEGACY/device_lenovo_A6020/commit/f16c24f22ec6dc03f05e44056fde4c1899b1135a)
> I'll merge the "A6020: Add exFAT and NTFS filesystem tools for AOSP builds" commit soon.

Just dropped the unnecessary commits, should be fine now!
